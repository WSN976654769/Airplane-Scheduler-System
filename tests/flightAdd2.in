location add Berlin 52.5 13.15 0.22222
location add NewYork 40.7 -74.26 -0.874
flight add sunday 20:00 Berlin NewYork 250
flights
flight 0
flight add monday 05:00 newYork berlin 234
location import locations.csv
FLIGHT ADD Tuesday 15:20 Sydney beijing 20 0
FLIGHT ADD Thursday 1:20 Sydney beijing 20 0
FLIGHT ADD Friday 1:20 Sydney beijing 20 0
FLIGHT ADD Saturday 1:20 Sydney beijing 20 0
FLIGHT 0 book -1
exit