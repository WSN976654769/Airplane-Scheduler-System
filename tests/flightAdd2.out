User: Successfully added location Berlin.

User: Successfully added location NewYork.

User: Successfully added Flight 0.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
   0 Sun 20:00   Mon 04:52   Berlin --> NewYork

User: Flight 0
Departure:    Sun 20:00 Berlin
Arrival:      Mon 04:52 NewYork
Distance:     6,387km
Duration:     8h 52m
Ticket Cost:  $1636.01
Passengers:   0/250

User: Scheduling conflict! This flight clashes with Flight 0 arriving at NewYork on Monday 04:52.

User: Imported 49 locations.
2 lines were invalid.

User: Successfully added Flight 1.

User: Successfully added Flight 2.

User: Successfully added Flight 3.

User: Successfully added Flight 4.

User: Invalid number of passengers to book.

User: Application closed.
